client/src/apis/navigationApi/navigationApi.js

```diff
@@ -0,0 +1,56 @@
+import { matchPath } from 'react-router-dom';
```

Comment:
Delete this file. I deleted it, but it appeared here because of not proper resolve of merge conflict